eve/io/mongo/validation.py

```diff
@@ -71,10 +71,10 @@ def _remove_unique_rules_on_fields_with_unique_index(self, schema):
         # removes the unique constraint on _id. We could use the information
         # available by app.data.driver.db[datasource].index_information() to
         # remove all unnecessary unique constraints.
-        result = copy.deepcopy(schema)
```

Comment:
@nam4dev i know but two things kept me from doing so:
1. unneeded function call to copy an empty dict
(but further more)
2. unneeded deepcopy of schema even if schema['_id']['unique'] does not exists.
This could be very expensive operation if schema is HUGE and that nested key does not exist.